<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GVSpectrogramWDialogSettings</class>
 <widget class="QDialog" name="GVSpectrogramWDialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrogram settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Window</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>identical to Spectrum settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbWindowSizeForcedOdd">
        <property name="text">
         <string>Force odd window size</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Window type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbSpectrogramWindowType">
          <item>
           <property name="text">
            <string>Hann</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hamming</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nutall</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman-Nutall</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman-Harris</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Flat top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rectangular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gaussian</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Exponential</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Generalized Normal</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="lblWindowNormPower">
              <property name="toolTip">
               <string>Power of the exponential's argument</string>
              </property>
              <property name="text">
               <string>power=</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spWindowNormPower">
              <property name="toolTip">
               <string>Power of the exponential's argument</string>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblWindowNormSigma">
            <property name="text">
             <string>sigma=</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spWindowNormSigma">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="lblWindowExpDecay">
          <property name="toolTip">
           <string>Targeted decay over half of the window length</string>
          </property>
          <property name="text">
           <string>Decay=</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spWindowExpDecay">
          <property name="toolTip">
           <string>Targeted decay over half of the window length</string>
          </property>
          <property name="suffix">
           <string> dB</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>60.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Short-Time Fourier Transform (STFT)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>identical to Spectrum settings</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Step size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sbStepSize">
          <property name="suffix">
           <string>s</string>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>0.000010000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Window size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sbWindowSize">
          <property name="suffix">
           <string>s</string>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.030000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Oversampling factor &lt;span style=&quot; font-size:8pt;&quot;&gt;[ dftlen=2^(ceil(log2(winlen))+X) ]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbSpectrogramOversamplingFactor">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>View</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Color map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbSpectrogramColorMaps"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSpectrogramColorMapReversed">
        <property name="text">
         <string>Reversed color map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAddMarginsOnSelection">
        <property name="text">
         <string>Add margins around selection when zoom on selection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
